### PR TITLE
[SHELL32] Distinguish floppy and removable drives

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

Distinguish floppy drive and non-floppy removable drive in icon and description.

JIRA issue: [CORE-10221](https://jira.reactos.org/browse/CORE-10221)

after

### CODE REVIEW

File: dll/win32/shell32/dialogs/drvdefext.cpp [hunk 1]
```diff
@@ -278,6 +283,115 @@ CDrvDefExt::PaintStaticControls(HWND hwndDlg, LPDRAWITEMSTRUCT pDrawItem)
...
+    IO_STATUS_BLOCK IoStatusBlock;
+    FILE_FS_DEVICE_INFORMATION DeviceInfo;
+
+    ntdll = GetModuleHandleA("ntdll");
+    if (ntdll == NULL)
+        return FALSE;
+
+    pNtQueryVolumeInformationFile =
+        (FN_NtQueryVolumeInformationFile)
+            GetProcAddress(ntdll, "NtQueryVolumeInformationFile");
```
Comment (learn-more) [inline dll/win32/shell32/dialogs/drvdefext.cpp:305]: We can probably call this directly here, without the need for GetProcAddress?

Comment (katahiromz) [reply dll/win32/shell32/dialogs/drvdefext.cpp]: OK.

**Comment by katahiromz** [comment]
Can you test, somebody? I don't have removable media.

**Comment by binarymaster** [comment]
@Extravert-ir I remember once you told to distinguish drive types, PnP-aware disk driver is required, can you check it?

**Comment by yagoulas** [comment]
I think I had a look at this some months ago. I remember seeing that GetDriveTypeW in ros doesn't return the correct type and when it is fixed we can use it to keep things clear. Could you have a look at it and if it works properly in ros?

**Comment by katahiromz** [comment]
I think the return value `DRIVE_REMOVABLE` of `GetDriveTypeW` is vague in Windows, between floppy drive type and non-floppy removable drive type.

**Comment by katahiromz** [comment]
I cannot contact the reporter of this issue. I cannot test this PR.

**Comment by katahiromz** [comment]
Assuming correct...

**Pull request merged**